oct_converter/readers/fda.py

```diff
@@ -186,9 +198,17 @@ def __init__(self, filepath):
             "size" / Int32un,
         )
 
+        self.align_info_header = Struct(
```

Comment:
we have started to move all the structs into their own files, under `readers/binary_structs` - there is an example for boct currently, and import them from there. I think doing this here will keep the FDA reader more readable and reduce the number of class attributes

Reply:
Okay I'll try to make files like that.